apps/editing-toolkit/editing-toolkit-plugin/help-center/class-wp-rest-help-center-search.php

```diff
@@ -73,24 +68,20 @@ public function get_search_results( \WP_REST_Request $request ) {
 		$query  = $request['query'];
 		$locale = $request['locale'];
 
-		if ( $this->is_wpcom ) {
-			$response = \WPCOM_Help_Search::search_wpcom_support( $query, $locale );
-		} else {
-			$query_parameters = array(
-				'query'  => $query,
-				'locale' => $locale,
-			);
-			$body             = Client::wpcom_json_api_request_as_user(
-				'/help/search/wpcom?' . http_build_query( $query_parameters )
-			);
-
-			if ( is_wp_error( $body ) ) {
-				return $body;
-			}
+		$query_parameters = array(
+			'query'  => $query,
+			'locale' => $locale,
+		);
+		$body             = Client::wpcom_json_api_request_as_user(
+			'/help/search/wpcom?' . http_build_query( $query_parameters )
+		);
 
-			$response = json_decode( wp_remote_retrieve_body( $body ) );
+		if ( is_wp_error( $body ) ) {
+			return $body;
 		}
 
+		$response = json_decode( wp_remote_retrieve_body( $body ) );
+
 		return rest_ensure_response(
 			array(
 				'wordpress_support_links' => $response,
```

Comment:
You removed the `wordpress_support_links` from `help-center-search-results.tsx`, but this is still adding it to the response.